src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,62 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private List<String> values;
+    private List<String> reports = new ArrayList<>();
```

Comment:
please don't use fields, use local variables
remember that your methods have return value and parameters where you can pass data